tests/plugins/multi-phase-plugin-a/src/lib.rs

```diff
@@ -42,20 +42,20 @@ impl HttpHandlers for AlicePlugin {
         if let Some(overwrite) = labels.get("overwrite") {
             assert_eq!(overwrite.as_str(), "new value");
         }
-        if response.status() == StatusCode::OK && request.method() == "POST" {
+        if response.status() == http::StatusCode::OK && request.method() == "POST" {
```

Comment:
Only the files within the SDK crate and within any code that uses the SDK are relevant. The admin endpoint uses the `StatusCode` type but doesn't go through the SDK to do it, so that's OK.